bunsen/bunsen-core/src/main/java/com/cerner/bunsen/definitions/StructureDefinitions.java

```diff
@@ -32,11 +32,25 @@ public abstract class StructureDefinitions {
       .add("positiveInt")
       .add("base64Binary")
       .add("uri")
+      // TODO: Figure out why these are added to R4 resource definitions.
```

Comment:
what happens if we remove the equivalent non-URL-prefixed types from the set? For example, what happens if we remove the entry `      .add("string")` and keep just `      .add("http://hl7.org/fhirpath/System.String")`

Reply:
We can't do that because this code is supposed to support both DSTU3 and R4 (and any other FHIR version we want to support). The problem is that string type in DSTU3 is represented by `string` while in R4 it is `System.String`.